### ISSUE TITLE
Add to helm repo https://charts.vectorized.io

### ISSUE THREAD

**Issue by vuldin** [redpanda-data/helm-charts#55]
Now that we have brought back and are actively improving this chart, we should improve on ease-of-use by adding it to the helm repo. At the moment users must clone the repo in order to use, which is more involved than:

```sh
> helm add redpanda/redpanda
```

This will also facilitate getting this chart into other tools such as [arkade](https://arkade.dev), [artifacthub](https://artifacthub.io), and others.

The repo already has the operator-based helm chart:

```sh
> helm search repo redpanda
NAME                            CHART VERSION   APP VERSION     DESCRIPTION                 
redpanda/redpanda-operator      v22.1.3         v22.1.3         Redpanda operator helm chart
```

This gets updated with the following CI workflow, which can be copied and modified slightly for this chart: https://github.com/redpanda-data/redpanda/actions/workflows/helm-chart-release.yml

I assume the chart name would be `redpanda/redpanda`.

**Comment by vuldin** [comment]
There is an existing CI workflow for this which needs to be ran once the chart is ready: https://github.com/redpanda-data/helm-charts/blob/main/.github/workflows/release.yaml

**Comment by vuldin** [comment]
Closing as a duplicate of #14
